README.md

```diff
@@ -107,7 +97,7 @@ To install this project, run
 

```

Comment:
![image](https://github.com/Bestbynature/Morse-Code-Interpreter/assets/93039359/e66ea3bf-9b8b-4e12-9af1-f7dd9cffd0f5)
**[OPTIONAL]**

- [ ] It would be nice if you remove one of the **back to top** buttons from the `Key Features` section to make your readme descriptive and professional :+1:

Reply:
Thank you so much @HAMMAS-SALEEM